m4_sessions/1711-inning/Group_4.md

```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
 
 Session Outline: Include the flow of the session and the time for each section.
 
-  * WHY - why it's important to write a technical blog post
-  * WHAT - topic of the blog
-  * HOW - how to write a technical blog (provide them a list of guidelines that will allow them to start writing that day)
-  * WHEN & WHERE - When and where to promote your blog.
+ * WHY it's important to write a technical blog post [MIMI] - 10 minutes
+  - Self-promotion
+  - Contribute/Give back to the community
+  - Provide examples of various Turing alumni blog posts & where they are now
+ * WHAT [MIKE] - 5 minutes
+  - High level: In-depth explanation of a technology, language, or algorithm
```

Comment:
Really like this high, mid, and low.  Great opportunity for people to see where they can start to plug in to the blogs.